data/relup_bases.eterm

```diff
@@ -0,0 +1,31 @@
+%% -*- mode: erlang; -*-
+
```

Comment:
need another copy of this file for enterprise edition.
so far the file will be identical, but technically it can be different. 
that is, open source edition can roll faster than enterprise.

Reply:
Since the script takes the DB path as input, maybe in the EE repo we can change the path to the `relup_bases.eterm`?